### DISCUSSION TITLE
Re-name GitHub action virtual env `macos-latest` to something else

### DISCUSSION THREAD

**Discussion by logankilpatrick** [community/community#9052] [Actions] [answered]
Hey folks, we just ran into a problem here: https://github.com/JuliaML/MLDatasets.jl/issues/86 where we were under the impression our code was running on the latest macOS version since we specific `macOS-latest` in the OS matrix. But in reality, the latest is pointing to an old version of macOS: https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#github-hosted-runners

Would it be possible to either update this or add a note at the bottom of that section that mentions this appropriately?

**Comment by ethomson** [comment, marked as answer]
Hi @logankilpatrick - sorry for the confusion.  We're rolling out macOS 11 as the default now for `macos-latest`, and we'll update the docs when this is complete.

**Reply by logankilpatrick** [reply]
Any thoughts as to if it's worth mentioning things like this in the docs? I still find it a bit puzzling that macOS-latest wont point to the latest version.

**Reply by logankilpatrick** [reply]
Though not sure if adding it to the docs would help since it is technically already there, the is just misleading.
